scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamicError.scala

```diff
@@ -48,4 +48,8 @@ object ScalafmtDynamicError {
 
   case class UnknownError(cause: Throwable)
       extends ScalafmtDynamicError("unknown error", cause)
+
+  object ConfigVersionRequired
+      extends Error("Version will be required, can't replace with default")
```

Comment:
```suggestion
      extends Error("Version is required, it can't be replaced with default")
```
?

Reply:
there's a nuance here. the configuration file might contain a version so the user could find this error message misleading.

what i'd like to say is, we can't use `withRespectVersion` or `withDefaultVersion` (whether or not version is present in the configuration file) because they would allow a version to be missing.